Map: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 28002/28002 [00:03<00:00, 8370.74 examples/s]
Map: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 3500/3500 [00:00<00:00, 9149.09 examples/s]
Map: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 3500/3500 [00:00<00:00, 9159.29 examples/s]
Some weights of BertForSequenceClassification were not initialized from the model checkpoint at emilyalsentzer/Bio_ClinicalBERT and are newly initialized: ['classifier.bias', 'classifier.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
Traceback (most recent call last):
  File "/mnt/dados/gpt_ed_assistant/scripts/train.py", line 89, in <module>
    args = TrainingArguments(
           ^^^^^^^^^^^^^^^^^^
TypeError: TrainingArguments.__init__() got an unexpected keyword argument 'evaluation_strategy'
